package.json

```diff
@@ -7,9 +7,12 @@
     "url": "git@github.com:factorialco/mobx-rest-axios-adapter.git"
   },
   "license": "MIT",
-  "jest": { "preset": "ts-jest" },
+  "jest": {
+    "preset": "ts-jest"
+  },
   "dependencies": {
     "axios": "0.19.0",
+    "lodash": "^4.17.15",
```

Comment:
do we need lodash for this? let's try to keep this dependency out if possible 🙏

Reply:
done. Thanks